docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,83 @@
+*******************************************************
+Storage Quota at /home/
+*******************************************************
+Staying under your ``/home/<username>`` quota is vital in preventing issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
+
+Analyze disk usage
+=======================================================
+From a compute node, ``srun --pty /bin/bash``, run the following command from your ``/home/<username>`` directory: ::
+    
+ du -shc .[^.]* *
+
+This command will output the size of each file, directory, and hidden directory in your ``/home/<username>`` space, with the total of your ``/home`` directory being the last line of the output. After identifying the large files and directories, you can move them to the appropriate location, (g.g., ``/work`` for research) or you can back up and delete the files and directories if they are no longer required. An example output would look like: ::
+ 
+ [<username>@<host> directory]$  du -shc .[^.]* *
+ 39M	.git
+ 106M	discovery-examples
+ 41K	README.md
+ 3.3M	software-installation
+ 147M	total
+
+Utlilize /scratch and /work
+=======================================================
+For long-term research work storage, ``/work`` should be used. If your PI does not have space in ``/work`` setup, they can request ``/work`` using the `New Storage Space Request <https://bit.ly/NURC-NewStorage>`_. PIs can also request additional storage using the `Storage Space Extension Request <https://bit.ly/NURC-StorageExtension>`_. More details of ``/work`` and ``/scratch`` can be found at :ref:`discovery_storage`. 
+
+For temporary job files, you can utilize ``/scratch/<username>``. Please be mindful of the ``/scratch`` purge policy, which can be found on the `Research Computing Policy Page <https://rc.northeastern.edu/policy/>`_.
+
+Clean ~/.conda directory
+=======================================================
+.. note::
+  Conda environments are part of your research and should preferably be stored in your PI's ``/work`` directory. 
```

Comment:
I left a comment below similar to this but I see your point on cleaning this up. I just don't know if just referencing /work is preferable since it is PIs that have to request /work directories and we don't want people trying to just point to /work and submitting tickets about it not being able to write there.